modules/calib3d/src/usac/ransac_solvers.cpp

```diff
@@ -733,7 +733,9 @@ class Ransac {
             const bool is_prosac = params->getSampler() == SamplingMethod::SAMPLING_PROSAC;
             std::atomic_bool success(false);
             std::atomic_int num_hypothesis_tested(0), thread_cnt(0), max_number_inliers(0), subset_size, termination_length;
-            std::atomic<double> best_score_all(std::numeric_limits<double>::max());
+            std::atomic_int best_score_all(std::numeric_limits<int>::max());
```

Comment:
I propose to use float here. It has the same 4 bytes, but the conversion is more accurate and range is better too.

Reply:
replaced int with float

Reply:
there seems to be a warning if I do this: "warning: overflow in implicit constant conversion [-Woverflow]"